CodeGeneration/Sources/generate-swift-syntax/GenerateSwiftSyntax.swift

```diff
@@ -154,32 +153,37 @@ struct GenerateSwiftSyntax: ParsableCommand {
       }
     )
 
-    var errors: [Error] = []
-    DispatchQueue.concurrentPerform(iterations: fileSpecs.count) { index in
-      let fileSpec = fileSpecs[index]
-      do {
-        var destination = destination
-        for component in fileSpec.pathComponents {
-          destination = destination.appendingPathComponent(component)
+    await withTaskGroup(of: (URL, Error?).self) { group in
```

Comment:
I would add labels to the tuple here. That way you don’t need to do the pretty magical and ambiguous `.0` and `.1` below.

```suggestion
    await withTaskGroup(of: (url: URL, error: Error?).self) { group in
```

Reply:
Made this change locally and pushed. It's definitely more readable now.